### ISSUE TITLE
How can I auto reload all the involved modules when the source is changed?

### ISSUE THREAD

**Issue by rudaoshi** [hoh/reloadr#12]
I want to auto reload all the involved modules when the source is changed. 

How can I do this?

**Comment by ghost** [comment]
Yeah, I think this makes sense whenever I rename functions. I would simply put a @reload at my main function and wish it all works automatically. Module imports, on the other hand, are usually executed at import time once, not on function call.

In the end it's probably more sensible to write an external watcher that kills your process and respawns it whenever a file change has been detected (probably waiting a certain time or waiting for a file system flag).
